src/qibolab/platform.py

```diff
@@ -411,15 +411,17 @@ def create_coupler_pulse(self, coupler, start, duration=None, amplitude=None):
     # TODO Add RY90 and RY pulses
 
     def create_RX90_drag_pulse(self, qubit, start, relative_phase=0, beta=None):
+        """Create native RX90 pulse with Drag shape."""
         qubit = self.get_qubit(qubit)
         pulse = self.qubits[qubit].native_gates.RX90.pulse(start, relative_phase)
         if beta is not None:
-            pulse.shape = "Drag(5," + str(beta) + ")"
+            pulse.shape = Drag(rel_sigma=pulse.shape.rel_sigma, beta=beta)
         return pulse
 
     def create_RX_drag_pulse(self, qubit, start, relative_phase=0, beta=None):
```

Comment:
Is this method truly useful with `beta=None`?

Reply:
We can drop the default and make `beta` a required argument, which should make sense given that if the user asks for a drag pulse should provide also `beta`.